flask-backend/api/routes/search.py

```diff
@@ -0,0 +1,123 @@
+import sys
+import os
+import sqlite3 as sql
+from types import new_class
+from flask import Blueprint, jsonify, request
+import re
+
+
+
+
+dirname = os.path.abspath(os.path.dirname(__file__))
+
+keyword = Blueprint('keyword', __name__, url_prefix="/keyword")
+
+dirname = (os.path.dirname(__file__))
+print(dirname, " dirname")
+
+'''
+    DATA_PATH, PATH_TO_REPORT,PATH_TO_TSV are support variables.
+'''
+
+DATA_PATH = '../../../data/'
+PATH_TO_REPORT = '/report/report.txt'
+PATH_TO_TSV = 'tsv/'
+
+
+cases_data_path = os.path.join(dirname,DATA_PATH)
+
+
+'''
+    checkword() is a bool function which returns true if
+    keyword found in path given else returns false.
+'''
+
+def checkword(pathname, keyword):
+
+    open_path_file = open(pathname, "r")
+    read_path_file = open_path_file.read()
+
+
+    if keyword in read_path_file:
+        open_path_file.close()
+        return True
+
+    open_path_file.close()
+
+    return False
+
+'''
+    searchkeyword() iterates over the data and checks if the keyword
+    is present in that file or not via checkword() function.
+
+'''
+def searchkeyword(keyword):
+
+    case_list = []
+    #print(cases_data_path," case_data_path")
+
+    for subdir, dirs, files in os.walk(cases_data_path):
```

Comment:
Sir @shivanshu1333, keyword search sould be done on all the cases inside data folder Or on only those cases that are related to a specific Admin and it's members ?

Reply:
Can you please elaborate on what do you mean by Admin, and it's members

Reply:
Sir He is trying to say that searching keyword in all the cases which belongs to a particular admin, not in all cases.

Reply:
I was saying that as of now each extracted case has a column for the I'd of the extractor who extracted that case.
Using this, each Admin has access to the list of cases that it's extractor member has extracted. So, my questions was the keyword feature should search over the cases that is included to an admin's extracted cases list or it should search to all the cases that is stored in server. 
PS:- @swapnalshahil has implemented this route in such a way that keyword is searching in all the cases that is stored in database.

Reply:
Searching only on the cases specific to admin will lead to a problem.
Let's suppose there are 26 admins working on case A to Z respectively. Now admin of case x wants to know if Mr.X (culprit) is involved with another cases or not?? So then admin doing case study of case X will have to ask other 25 admins if there they know Mr.X. This will create a problem. That's why I think this method is pointless.

Reply:
Searching a keyword in all the cases stored in database makes more sense

Reply:
That is what I did.